mlflow/java/client/src/main/java/org/mlflow/artifacts/CliBasedArtifactRepository.java

```diff
@@ -140,6 +140,22 @@ public File downloadArtifacts() {
     return listArtifacts(null);
   }
 
+  /**
+   * Only available in the CliBasedArtifactRepository. Downloads an artifact to the local
```

Comment:
Add experimental warning here too? Since the class and its methods do show up public docs (e.g. https://mlflow.org/docs/0.5.2/java_api/org/mlflow/artifacts/CliBasedArtifactRepository.html)

Reply:
Done